### ISSUE TITLE
Unsupported keras layer type Lambda (0.9.2-SNAPSHOT)

### ISSUE THREAD

**Issue by blackredscarf** [deeplearning4j/deeplearning4j#4730]
I run this code:
```
object InceptionRest {
  def main(args: Array[String]) {
    val conf: ComputationGraphConfiguration =
      KerasModelImport.importKerasModelConfiguration("update.json")
    val net: ComputationGraph = new ComputationGraph(conf)

    println(net.summary())
  }
}
```
And get this error:
```
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unsupported keras layer type Lambda. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:259)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:158)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:92)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:123)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelConfiguration(KerasModelImport.java:272)
	at pers.zjw.dl4j.snapshot.keras2.InceptionRest$.main(InceptionRest.scala:15)
	at pers.zjw.dl4j.snapshot.keras2.InceptionRest.main(InceptionRest.scala)
```

the Lambda layer in keras2:
```
def scaling(x, scale):
    return x * scale
```
```
up = Lambda(scaling,
			output_shape=K.int_shape(up)[1:],
			arguments={'scale': scale})(up)
```

how can I solve it ? thanks.

**Comment by maxpumperla** [comment]
@blackredscarf

**Comment by blackredscarf** [comment]
@maxpumperla 
Sorry, I'm not familiar with custom keras layers. How do I get `output_shape` and` arguments` in Lambda? And do I need to write a custom layer to handle it?

This looks a bit cumbersome. Whether or not there is an API like Keras allows me to focus on functions without having to do anything else

**Comment by maxpumperla** [comment]
@blackredscarf yes, you'd need to write a layer for this as it is right now. we plan to support Lambdas in the same fashion as keras does at some point. Right now, we're working on our autodiff module SameDiff in ND4J, which would ultimately take care of the gradients for you, so you "don't have to do anything else".

If you're interested, here's a layer to support import of a custom space to depth Lambda from keras:

https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/layers/convolutional/KerasSpaceToDepth.java#L39-L42

Sorry, I can't do much more for you right now.
